sql/core/benchmarks/InExpressionBenchmark-results.txt

```diff
@@ -2,550 +2,739 @@
 In Expression Benchmark
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
```

Comment:
Recently, https://github.com/apache/spark/pull/23914 added `Stdev` to the benchmark result. We need to rerun this.

@aokolnychyi . After you update the PR code, I'll rerun the benchmark on EC2 and make a PR to you.